### PR TITLE
test: CXTB-374 added call participant call missed

### PR DESCRIPTION
**Pull request by miguel-hquez**
Automated test case:
https://digitalscientists.atlassian.net/browse/CXTB-128

- Added test.
- Added page objects.

Proof of Successful run:
image

### CODE REVIEW

**Review by csanchez-nts — DISMISSED** [review]

File: tests/cases/care_partner_platform/case_care_platform_participants.yaml [hunk 1]
```diff
@@ -1524,4 +1524,218 @@ TestCarePartnerCallParticipantsDeviceCallIgnored:
...
+      Greps:
+        - var: PARTICIPANT_NOT_AVAILABLE_TEXT
+          attr: textContent
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Asserts:
+        - Type: eq
+          Var: PARTICIPANT_NOT_AVAILABLE_TEXT
+          Value: 'Miguel is not available.Would you like to leave a text message?'
```
Comment (dguzmanneural) [inline tests/cases/care_partner_platform/case_care_platform_participants.yaml:1621]: Hardcoded message

Comment (miguel-hquez) [reply tests/cases/care_partner_platform/case_care_platform_participants.yaml]: commit [ba07e01](https://github.com/neveralone-chs-org/TestRay/pull/99/commits/ba07e01c87c60fac0169388131c34d872eb65fa7) has the fix

**Review by csanchez-nts — APPROVED** [review]

File: tests/cases/care_partner_platform/case_care_platform_participants.yaml [hunk 2]
```diff
@@ -1524,4 +1524,218 @@ TestCarePartnerCallParticipantsDeviceCallIgnored:
...
+  # Calling Login case.
+    - Type: case
+      Value: CarePartnerLogin
+  # Navigate to Participants.
+    - Type: case
+      Value: CarePartnerNavigateToParticipants
+  # Click on the corresponding participant.
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.senior_complete_name$
```
Comment (dguzmanneural) [inline tests/cases/care_partner_platform/case_care_platform_participants.yaml:1557]: Please check this page element, the component has changed.

**Review by dguzmanneural — APPROVED** [review]

File: tests/cases/care_partner_platform/case_care_platform_participants.yaml (hunk 2 above)
Comment (miguel-hquez) [reply tests/cases/care_partner_platform/case_care_platform_participants.yaml:1557]: https://github.com/neveralone-chs-org/TestRay/pull/101 this PR contains the update for the element

**Pull request merged**